docs/overview/header.md

```diff
@@ -5,3 +5,74 @@ to the top of the viewport even when the main content area is scrolled up and do
 In order to create a header you have to create a CDI bean that implement org.uberfire.client.workbench.Header interface.
 
 ![header](header.png)
+
+Let us add a simple Composite that adds 3 labels. 
+
+In package org.uberfire.client.navbar we can create 
+
+- HeaderView.java
+
+```
+package org.uberfire.client.navbar;
+
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.HorizontalPanel;
+import com.google.gwt.user.client.ui.Label;
+
+public class HeaderView extends Composite {
+
+    public HeaderView() {
```

Comment:
It's a good example, but we suggest user for this example MVP pattern (presenter + view) and also create the header with Errai UI (templates). In that way, we can teach our users to use our own stack!